CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.7.6
```

Comment:
Is the convention that we make an `Unreleased` commit first and then make another commit to bump the version?

Reply:
I was thinking this. I've been looking back at some old PRs and I reckon that convention makes sense. I'll update accordingly.